api/src/main/java/org/pmiops/workbench/exceptions/GatewayTimeoutException.java

```diff
@@ -0,0 +1,28 @@
+package org.pmiops.workbench.exceptions;
+
+import org.pmiops.workbench.model.ErrorResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.GATEWAY_TIMEOUT)
+public class GatewayTimeoutException extends WorkbenchException {
+  public GatewayTimeoutException() {
+    super();
+  }
+
+  public GatewayTimeoutException(String message) {
+    super(message);
+  }
+
+  public GatewayTimeoutException(ErrorResponse errorResponse) {
+    super(errorResponse);
+  }
+
+  public GatewayTimeoutException(Throwable t) {
+    super(t);
+  }
+
+  public GatewayTimeoutException(String message, Throwable t) {
+    super(message, t);
+  }
```

Comment:
I would leave out the unused methods here and only implement what you need, the first one.